src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -55,12 +53,14 @@ func NewClient(addr, appName string) *MobileClient {
 
 	hqfd := client.Configure(clientConfig.Client)
```

Comment:
I agree though we could just initialize it when it's declared. I went ahead and made that change.